controller/app.go

```diff
@@ -99,20 +101,17 @@ func (r *AppRepo) Add(data interface{}) error {
 
 func scanApp(s postgres.Scanner) (*ct.App, error) {
 	app := &ct.App{}
-	var meta hstore.Hstore
+	var meta []byte
 	var releaseID *string
 	err := s.Scan(&app.ID, &app.Name, &meta, &app.Strategy, &releaseID, &app.CreatedAt, &app.UpdatedAt)
 	if err == sql.ErrNoRows {
-		err = ErrNotFound
+		return nil, ErrNotFound
```

Comment:
This isn't the only error that can be returned from `Scan`, probably should add an `else if err != nil`